packages/block-library/src/rss/style.scss

```diff
@@ -1,5 +1,8 @@
 .wp-block-rss {
-	box-sizing: border-box !important; // The important is needed because of the reset mixin on the parent: https://github.com/WordPress/gutenberg/blob/a250e9e5fe00dd5195624f96a3d924e7078951c3/packages/edit-post/src/style.scss#L54
+	// This needs extra specificity due to the reset mixin on the parent: https://github.com/WordPress/gutenberg/blob/a250e9e5fe00dd5195624f96a3d924e7078951c3/packages/edit-post/src/style.scss#L54
+	&.wp-block-rss {
+		box-sizing: border-box;
```

Comment:
I prefer this much more than !important. I wonder why the theme css is not loaded after the editor's, wouldn't that solve all these kinds of issues?

Reply:
Yeah, I prefer the double-selector approach here too. 👍

Reply:
It seems the extra specificity is only needed in the frontend, so I wonder whether we should keep "style" pure and add the specificity in "editor.scss". Not a big deal though.

Reply:
IMO the benefits of having the same code in the editor and frontend are greater so I'd prefer to keep it like this.